updating/updating-cluster-within-minor.adoc

```diff
@@ -22,9 +22,10 @@ See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permis
 * Ensure all Operators previously installed through Operator Lifecycle Manager (OLM) are updated to their latest version in their latest channel. Updating the Operators ensures they have a valid update path when the default OperatorHub catalogs switch from the current minor version to the next during a cluster update. See xref:../operators/admin/olm-upgrading-operators.adoc#olm-upgrading-operators[Upgrading installed Operators] for more information.
 * Ensure that all machine config pools (MCPs) are running and not paused. Nodes associated with a paused MCP are skipped during the update process. You can pause the MCPs if you are performing a canary rollout update strategy.
 //remove this???^ or maybe just add another bullet that you can break up the update?
-* To accommodate the time it takes to update, you are able to do a partial update by updating the worker or custom pool nodes. You can pause and resume within the progress bar of each pool. 
+* To accommodate the time it takes to update, you are able to do a partial update by updating the worker or custom pool nodes. You can pause and resume within the progress bar of each pool.
```

Comment:
I am confused as to what this sentence is trying to say. 
The partial update usually refers to only updating the control plane nodes while the worker pools are paused. Is this trying to say that you can customize which workers are updated using custom pools to only pause specific worker updates?

Reply:
@sanchezl This isn't something that I updated at all, it was already in this file (my update just removed whitespace at the end of this line). So I can't speak to what it means, but I don't think we need to touch it for this PR